### PR TITLE
improving the readme

### PR DESCRIPTION
**Pull request by sadiqevani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,37 +138,27 @@ Name it after what users will see, not what it represent in your application.
...
-        └── aeComponentName
-            ├── aeComponentName.js
-            ├── aeComponentName.md
-            ├── aeComponentName.scss
-            ├── aeComponentName.vue
-            └── index.js
+        └── ae-component-name
+            ├── ae-component-name.md
+            ├── ae-component-name.test.js
+            └── ae-component-name.vue
```
Comment (davidyuk) [inline README.md:157]: I still don't agree with kebab case in component naming. I thought that we are using it to distinguish new and old components

File: README.md [hunk 2]
```diff
@@ -320,10 +348,11 @@ new Vue({
 </template>
 
 <script>
-import { AeMain, aeHelperMixin } from '@aeternity/aepp-components'
+import { AeMain, mixins } from '@aeternity/aepp-components'
 
 export default {
   name: 'app',
+  mixins: [mixins.events],
```
Comment (davidyuk) [inline README.md:355]: Is tree shaking works in this case? Is unused mixins will be removed from the bundle?

File: README.md [hunk 3]
```diff
@@ -8,17 +8,12 @@ navigation: 3
...
 
 The github repository is [here](https://github.com/aeternity/aepp-components).
 
-## Hackathon Notice
-We've rewritten and refactored the components for better usage in the hackathon (24.11.2017). 
-Beware that the currently existing apps (like identity, aexistence/notary, wall, et cetera) 
-still use older versions of this repository. In the near future we will update these repositories 
-to use the latest aepp-components.
+__Note__: this repository is currently in active development and things might break / change in the future
+but we will be adhering to the Semantic Versioning.
```
Comment (davidyuk) [inline README.md:16]: Can we be compatible with semantic versioning in current development?

**Review by davidyuk — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (davidyuk) [reply README.md:157]: We have a talk with Emil in September and as I understand we decided to revert kebab-case and use CamelCase as it was before.

Comment (sadiqevani) [reply README.md:157]: > We have a talk with Emil in September and as I understand we decided to revert kebab-case and use CamelCase as it was before.

I remember differently, we agreed to 90% of your requests, but the naming of the components we agreed to keep it kebab-case.

File: README.md (hunk 2 above)
Comment (sadiqevani) [reply README.md:355]: This is a README.md file, its just an example! Not sure where tree-shaking is supposed to work here?

File: README.md (hunk 3 above)
Comment (sadiqevani) [reply README.md:16]: Thats why this PR exists: https://github.com/aeternity/aepp-components/pull/210 after this we'll start publishing in the main `@aeternity/aepp-components` package with semantic versioning as you're doing in the `@aeternity/aepp-components-3` package.

File: README.md (hunk 2 above)
Comment (davidyuk) [reply README.md:355]: looks like a bad example

File: README.md (hunk 1 above)
Comment (ricricucit) [reply README.md:157]: This looks fine to me. I don't see any big cons in using kebab-case naming for file names.

File: README.md (hunk 2 above)
Comment (ricricucit) [reply README.md:355]: I've created an [issue to test tree-shaking](https://github.com/aeternity/aepp-components/issues/214). please, @sadiqevani don't forget to test this.

File: README.md (hunk 3 above)
Comment (ricricucit) [reply README.md:16]: I don't see the issue with adhering with sematic versioning in the (very near?) future.

**Review by ricricucit — APPROVED** [review]
I understand some of these changes are mentioning things that might not work or be unclear ATM (tree-shaking, semver) and I've created issue #214 and #215 to discuss non-blocking issue in a 2nd step.

**Pull request merged**